### PR TITLE
Update BGE Batch labels for CC processing

### PR DESCRIPTION
**Pull request by sokosaurus**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.html [hunk 1]
```diff
@@ -52,6 +52,13 @@ <h1 class='slds-text-body_regular'>{CUSTOM_LABELS.commonPaymentServices}</h1>
                         </div>
                     </lightning-layout-item>
 
+                    <lightning-layout-item size='12' 
+                                           padding='around-small' 
+                                           if:true={shouldDisplayCardProcessingGuidanceMessage}>
+                        <div class='slds-text-body_regular'>
+                            <b>The donor's credit card will be processed when you process the batch.</b>
```
[the remark this answers is not in the document]
Comment (sokosaurus) [reply src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.html:59]: Absolutely, thanks for calling this out! I'm planning to get this updated shortly.

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -5085,6 +5085,13 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Button to process a batch.</shortDescription>
         <value>Process Batch</value>
     </labels>
+    <labels>
```
Comment (sokosaurus) [inline src/labels/CustomLabels.labels:5088]: @melissabarber - I've added these two labels to update the "Process Batch" button (top label) and to add a new block of text to the Elevate widget in batch mode (bottom label). Full details of these labels can be found in the AC here, https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000094E17IAE/view.

Do these labels and their names work or should we make any adjustments here?

File: src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.html [hunk 3]
```diff
@@ -10,10 +10,11 @@
                                   data-action={ACTIONS.DRY_RUN_BATCH}
                                   data-qa-locator={qaLocatorBatchDryRun}>
                 </lightning-button>
-                <lightning-button label={LABELS.bgeProcessBatch}
+                <lightning-button label={processBatchButtonName}
                                   onclick={handleClick}
                                   data-action={ACTIONS.PROCESS_BATCH}
-                                  data-qa-locator={qaLocatorProcessBatch}>
+                                  data-qa-locator={qaLocatorProcessBatch}
+                                  aria-label={processBatchButtonAltName}>
```
Comment (sokosaurus) [inline src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.html:17]: Hey @bdvorachek - Hoping you might be the best person to connect with on this. My work item (https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000094E17IAE/view) calls to add "alt-text" to the "Process Batch" button here for the purposes of supporting screen readers.

In my research, it looks like the LWC "lightning-button" component doesn't support this, so I'm using "aria-label" instead. Do you know if this is the ideal workaround or would you happen to have another recommendation on the best way to move forward here? Thanks!

File: src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.js [hunk 4]
```diff
@@ -62,4 +62,17 @@ export default class GeBatchGiftEntryHeader extends LightningElement {
...
+        let buttonName = this.LABELS.bgeProcessBatch;
+        if (this.batchTotals.authorizedPaymentsCount) {
+            buttonName = this.LABELS.bgeProcessBatchAndPayments;
+        }
+        
+        return buttonName;
+    }
+
+    get processBatchButtonAltName() {
+        return `${this.processBatchButtonName} button`;
```
Comment (sokosaurus) [inline src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.js:76]: Also, @melissabarber - one final question.. do you think we need to localize the word "button" on the alt-text for the "Process Batch" button. Right now, the ask is to have the button alt-text be either:

Process Batch button 

OR

Process Batch and Payments button 

In both scenarios, everything leading up to the word "button" would be localized and I'm wondering if it makes sense to ensure that _everything_ is localized here. Thanks in advance for your feedback!

File: src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.html (hunk 3 above)
Comment (bdvorachek) [reply src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.html:17]: I think `aria-description` or `aria-describedby` is more appropriate if we're providing a longer-form description of what the button does. I'd have to look at the behavior in a screen reader, but I think `aria-label` ends up clashing with the regular `label` attribute.

File: src/labels/CustomLabels.labels [hunk 5]
```diff
@@ -5085,6 +5085,13 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
...
+        <fullName>bgeProcessBatchAndPayments</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button to process a batch that contains Elevate payments.</shortDescription>
+        <value>Process Batch and Payments</value>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels:5093]:
```suggestion
        <value>Process Batch & Payments</value>
```

Comment (melissabarber) [reply src/labels/CustomLabels.labels:5093]: FYI @laurenchen-sf updated "and" to "&"

File: src/labels/CustomLabels.labels [hunk 6]
```diff
@@ -5085,6 +5085,13 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Button to process a batch.</shortDescription>
         <value>Process Batch</value>
     </labels>
+    <labels>
+        <fullName>bgeProcessBatchAndPayments</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button to process a batch that contains Elevate payments.</shortDescription>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels:5092]:
```suggestion
        <shortDescription>Button to process a batch and payments through {0}</shortDescription>
```

Comment (melissabarber) [reply src/labels/CustomLabels.labels:5092]: {0} = commonPaymentServices

**Review by melissabarber — COMMENTED** [review]
@sokosaurus added a couple of suggestions for the new button.

**Review by voduyemi — COMMENTED** [review]
Looks good to me! @sokosaurus pending your accessibility changes.

**Review by ptewson-sfdo — APPROVED** [review]
This looks good - my feedback is already captured in comments from others!

**Pull request merged**